cluster-provision/k8s/1.17/provision.sh

```diff
@@ -2,6 +2,12 @@
 
 set -ex
 
+# Configure IPv6 DHCP
+nmcli connection modify "System eth0" ipv6.method dhcp
+nmcli connection modify "System eth0" ipv6.may-fail no
+nmcli connection modify "System eth0" connection.autoconnect yes
+nmcli connection down "System eth0" && sudo nmcli connection up "System eth0"
+
```

Comment:
is 
nmcli connection modify "System eth0" ipv6.addr-gen-mode eui64
should be added here  as Roman said ?

Reply:
Is now there.